### ISSUE TITLE
Accuracy instead of Dice-coeff

### ISSUE THREAD

**Issue by Rabia-Metis** [jocicmarko/ultrasound-nerve-segmentation#61]
Thanks for the amazing work
Can you kindly tell me what will be the customised function for accuracy. I want to use accuracy as a metric instead of dice-coeff

**Comment by NirDiamant** [comment]
You can define your own loss for accuracy, for example by pixel comparison. I beleive it will be a good solution because you can define the thresholding by yourself

**Comment by mrgloom** [comment]
Maybe just add metric?
`model.compile(optimizer=Adadelta(), loss=dice_coef_loss, metrics=[dice_coef,my_custom_metric])`
